pkg/kubefed2/disable.go

```diff
@@ -298,12 +299,45 @@ func generatedFederatedTypeConfig(config *rest.Config, enableTypeDirective *enab
 }
 
 func deleteFederatedType(config *rest.Config, typeConfig typeconfig.Interface, write func(string)) error {
+	federatedTypeAPIResource := typeConfig.GetFederatedType()
+	crdName := typeconfig.GroupQualifiedName(federatedTypeAPIResource)
+	exists, err := federatedTypeCustomResourcesExist(config, &federatedTypeAPIResource)
+	if err != nil {
+		return err
+	} else if exists {
+		return errors.Errorf("Cannot delete CRD %q while resource instances exist. Please delete them and try again or remove --delete-crd option to disable only\n", crdName)
```

Comment:
I found the language a little confusing.
Consider rewording to "Please try `kubefed2 disable` again after removing the resource instances or without the option `--delete-CRD`"
Or something similar. The usage of the word disable is a little tricky, I think.

Reply:
Updated, PTAL.